File: beginner/C/countingPrime.c
#include<stdio.h>
#include<math.h>

int main(void){
    unsigned long long n = 0;
    const double ERRORAJUST = 1.25506;

    //input
    scanf("%lli", &n);

    //min
    double min = n / log(n);

    //max 
    double max = ERRORAJUST * min; //same value of n/log(n)


    printf("%.1f %.1f\n", min, max);

    return 0;
}

File: beginner/php/countingPrime.php
<?php

$n = trim(readline());

//minimun value
$min = $n / log($n);

//max value
$max = 1.25506 * ($min); //same value

//bug, falta o zero
echo round($min, 1) . ' ' . round($max, 1) . PHP_EOL;